ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/fragmentisers/geospatial/config/GeospatialConfig.java

```diff
@@ -27,11 +18,11 @@ public void setBucketiserProperty(String bucketiserProperty) {
     	this.bucketiserProperty = bucketiserProperty;
     }
     
-    public int getMaxZoomLevel() {
-    	return maxZoomLevel;
+    public int getZoomLevel() {
```

Comment:
Would definitely maintain the "max" zoom level. saying that we won't go any deeper into the tiles than this.

Reply:
Updated